env.example

```diff
@@ -2,4 +2,5 @@ PAGOPA_FUNCTIONS_CHECKOUT_HOST=localhost
 PAGOPA_FUNCTIONS_CHECKOUT_PORT=7071
 ENDPOINT_DELAY=0
 CHECK_STATUS_ADDITIONAL_ATTEMPTS=0
-NPG_APY_KEY=npg-api-key
+NPG_API_KEY=npg-api-key
+PSP_API_KEY=psp-api-key
```

Comment:
Add this to README :)

Reply:
Fixed with fc444281c7cb2133873b6f314b1d264b52bcfad6